en/docs/deploy-and-publish/deploy-on-gateway/api-microgateway/getting-started/quick-start-guide/quick-start-guide-docker-with-apim.md

```diff
@@ -0,0 +1,96 @@
+# Quick Start Guide with API Manager - Docker
+
+Lets' deploy an API from API Manager(control plane) and invoke via the Microgateway.
+
+### Before you begin..
+
+Make sure to install and set up [Docker](https://www.docker.com).
+
+### Step 1 - Download and setup Microgateway distribution zip
+
+Latest Microgateway distribution can be downloaded from the [page.](https://wso2.com/api-management/api-microgateway/) Extract the Microgateway distribution zip. Extracted folder will be called as `MG_HOME` hereafter.
+
+### Step 2 - Configure Microgateway to Connect to API Manager
+We have to enable the `[controlPlane.eventHub]` in the `config.toml` in the `MG_HOME/resources/conf` file. 
+Let's edit the `config.toml` like below to enable the event hub.
+
+``` java
+[controlPlane.eventHub]
+  enabled = true
+```
+
+
+### Step 3 - Start Microgateway and API Manager
+
+First we need to add the host entry to `/etc/hosts` file in order to access the API Manager publisher and dev portal.
+Add the following entry to `/etc/hosts` file
+
+``` java
+127.0.0.1   apim
+```
+
+
+Start the Microgateway and API Manager on docker by executing the docker compose script inside the `MG_HOME/docker-with-apim` folder.
+
+``` java
+docker-compose up -d
+```
+
+Once containers are up and running, we can monitor the status of the containers using the following command
+``` java
+docker ps | grep mg-
+```
+
+!!! info
```

Comment:
In "rich diff" mode, the "!!!info" and "!!!tip" are not shown in the preview as they are supposed to be. Shall we check this? May an indention might need.